Update of Service - email (#422)
SuperOfficeDocs/superoffice-docs: docs/email/Service.md

  </State>
</FeatureToggles>
```
</details>

#### How to use OAuth 2.0 in SuperOffice Service?

> Next time you create a mailbox in SuperOffice Service, add your Microsoft 365 (Microsoft® Exchange Online) email account, and you will be redirected to Microsoft for authentication:

Reviewer: ```suggestion
When a mailbox is created in SuperOffice Service, and a Microsoft 365 (Microsoft® Exchange Online) email account is added, you will be redirected to Microsoft for authentication:

```

For OAuth to be able to work, -make sure you have user-credentials (UPN) that directly authorize you on the mailbox, not the email address or a user which has access to it as a *shared mailbox*.
     
> [!NOTE]
> Up to version 10.1.4: If we don't recognize the UPN you will be redirected to mailbox properties. Click Cancel -> redirect back to mailboxes tab. Fix and verify the UPN. Make sure your MX Record is pointing to the Microsoft® Exchange Online server. Try again.

Click below to see details on UPN:
<details>

 **What is UPN (User Principal Name)?**

In Windows Active Directory, a User Principal Name (UPN) is the name of a system user in an email address format. A UPN (for example: john.doe@domain.com) consists of the user name (logon name), separator (the @ symbol), and domain name (UPN suffix).

A UPN is not the same as an email address. Sometimes, a UPN can match a user's email address, but this is not a general rule.

**How to check or modify the UPN of a user**

Use the links below to learn how to check and change UPNs in various environments.

* [Microsoft Exchange Server][6]
* [Office 365][7]
 
</details>

### MX Record pointing to the Microsoft® Exchange Online server

> We need to ensure that the MX record points to the Microsoft address.

Reviewer: ```suggestion
Ensure that the MX record points to the Microsoft address.

```

- How it looks from where NetServer is installed on the internal network, that applies:

To see this, follow the steps below.

1. Open "command".
2. Type nslookup and press Enter
3. Type set type=mx, then Enter.
4. Enter your domain
If the MX does not point to Microsoft, then this is the cause of the problem, and your IT administrator needs to resolve this DNS issue.

### Duplicate serial number
     

> If you get the following error when trying to setup OAuth:

Reviewer: ```suggestion
When the following error is observed when configuring OAuth:

```
